More evaluation metrics added

SinghShreya05 opened the pull request:
1. Previously evaluation-metrics consisted of "MASE", "sMAPE", "MAPE". Have added "RMSSE", "MAD", "GMAE", "RMSPE" to it.
2. These relevant evaluation metrics are - Root mean squared scaled error (RMSSE), Mean absolute deviation (MAD), Geometric Mean Absolute Error (GMAE), Root Mean Squared Percentage Error (RMSPE).
3. Added the docstring too.
4. Included these changes in "_functions.py" and "_classes.py" files.

RNKuhns commented:
@SinghShreya05 PR #801 also includes an update of  forecasting metrics. I believe it includes all of these except for GMAE. I believe we are getting close to finishing that PR.  Take a look at `_functions.py` in #801. It updates the naming conventions of forecasting metrics to match Scikit-learn (e.g. use mean_squared_error instead of mse_loss, etc) and also to include multioutput and weighting functionality in the Scikit-learn metrics.  

You could add GMAE using the new naming conventions used in #801 (e.g. geometric_mean_absolute_error); could also add the median version (e.g. geometric_median_absolute_error). 

If you have any questions about converting these to that format let me know.

SinghShreya05 commented:
@RNKuhns I have added "geometric_mean_absolute_error" and "geometric_median_absolute_error" to the "_functions", "_classes" and "__init__" files. I have converted my code to the desired format and maintained proper naming conventions. Please review the PR sent to the "new-forecasting-metrics" branch.

mloning commented:
Closed in favour of #837
